README.md

```diff
@@ -173,13 +170,4 @@ The Ray cluster used for computing allows several settings through the following
 | RAY_WORKER_GPUS | "" | Number of GPUs available for worker nodes. |
 | RAY_WORKER_GPUS_FRACTION | "" | Fraction of available GPUs used by worker nodes. |
 
-## Rebuilding dependencies
```

Comment:
suggestion: add a section on where to add dependencies, which is now per-requirements file?

Reply:
There's a section above on adding deps (using `uv add`)